src/index.html

```diff
@@ -165,7 +165,7 @@ <h2>Resize the window to switch between images</h2>
 
 	</div>
 
-	<script src="/scripts.js"></script>
+	<script src="/vanila-js-scss-seed-project/scripts.bundle.js"></script>
```

Comment:
`vanila-js-scss-seed-project` should not be part of the script src, we should use <base href/> to fix this, this should work out of the box, I will test it and if it dose not work, make it

Reply:
I debugged for some hours and I can't find why I need to add  the repository name into the path to scripts.bundle.js in order to work correctly.

Reply:
there is a trick in the webpack config that made it work before, `   {search: 'src="/scripts.js"', replace: `src="${baseHref}scripts.js"`},` you need to adjust this line in order to accommodate the changes to the file name

Reply:
I changed the path search and replace, now it is working correctly.